test/compress/template-string.js

```diff
@@ -558,3 +558,135 @@ baz`;
     ]
     node_version: ">=6"
 }
+
+tagged_template_with_invalid_escape: {
+    input: {
+        function x(s) { return s.raw[0]; }
+        console.log(String.raw`\u`);
+        console.log(x`\u`);
+    }
+    expect_exact: "function x(s){return s.raw[0]}console.log(String.raw`\\u`);console.log(x`\\u`);"
+    expect_stdout: [
+        "\\u",
+        "\\u",
+    ]
+    node_version: ">=10"
+}
+
+es2018_revision_of_template_escapes_1: {
+    options = {
+        defaults: true,
+    }
+    input: {
+        console.log(String.raw`\unicode \xerces`);
+    }
+    expect_exact: "console.log(String.raw\`\\unicode \\xerces\`);"
```

Comment:
Please use the following input as per [patch](https://github.com/terser-js/terser/issues/129#issuecomment-431836246):
```
        console.log(String.raw`\unicode \xerces \1234567890`);
```
It's valid ES2018 code.
```
$ echo 'console.log(String.raw`\unicode \xerces \1234567890`);' | node-v10.4.1
\unicode \xerces \1234567890
```

Reply:
Note that the invalid octal escaped is only permissible in ES2018 tagged template strings, not untagged template strings:
```
$ echo 'console.log(`\1234567890`);' | node-v10.4.1
[stdin]:1
console.log(`\1234567890`);
              ^^
SyntaxError: Octal escape sequences are not allowed in template strings.
```